bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup.Tests/KeyCeremony/TestKeyCeremony.cs

```diff
@@ -0,0 +1,91 @@
+using ElectionGuard.UI.Lib.Models;
+
+namespace ElectionGuard.ElectionSetup.Tests.KeyCeremony;
+[TestFixture]
+public class TestKeyCeremony : DisposableBase
+{
+
+    [Test]
+    public void Test_KeyCeremony()
```

Comment:
this is basically the same as the key ceremony step in the end to end election in python but since we are not trying to assert formal functional patterns, we do not implement it as a functional test. instead, we run the process using the mediator. this code is basically the same as is in the generator above except that it also asserts at each step whereas the generator does not. I considered wether to write this out in two places like that but i think it makes sense since the purpose of the generator is to create the data for use in downstream tests where as the purpose of this is to actually verify the steps.